polyschedule/schedule_api.py

```diff
@@ -1,18 +1,19 @@
 from typing import Tuple, Optional
+
 import aiohttp
 import orjson
+
 from polyschedule.exceptions import GettingScheduleError
 
 
-async def _request_schedule(group_id, date) -> str:
+async def _request_json(group_id, date) -> str:
```

Comment:
Всё же логичнее, наверное, так будет
```suggestion
async def _request_schedule_json(group_id, date) -> str:
```